game/main.cpp

```diff
@@ -201,6 +201,7 @@ int main(int argc, char** argv) {
           break;
         case RuntimeExitStatus::RESTART_IN_DEBUG:
           force_debug_next_time = true;
+          MasterExit = RuntimeExitStatus::RUNNING;
```

Comment:
can you try putting this on line 194, right before the `exec_runtime` call? I think that's the right place to put it.

I think this is potentially a race condition on startup that shows up because I disabled the VM stuff, which makes graphics stuff start up about 200ms faster now, so the graphics thread beats the EE thread in setting this flag. The right thing to do is set MasterExit to running before we start anything.

Reply:
Line moved.  Ah I saw the VM changes, but I was thinking it was unrelated.

Reply:
yeah I didn't expect it to make a difference either - I think this was always buggy and we just got really lucky with the timing before.